src/DIRAC/RequestManagementSystem/Agent/CleanReqDBAgent.py

```diff
@@ -148,12 +157,32 @@ def execute(self):
                 self.log.info("execute: deleting request '%s' with status %s" % (requestID, status))
                 delRequest = self.requestClient().deleteRequest(requestID)
                 if not delRequest["OK"]:
-                    self.log.error("execute: unable to delete request '%s': %s" % (requestID, delRequest["Message"]))
+                    self.log.error("execute: unable to delete request", "'%s': %s" % (requestID, delRequest["Message"]))
                     continue
                 deleted += 1
 
+        # optional: Set Scheduled requests to Cancelled if older than threshold
+        if self.cancelGraceDays > 0:
+            statusList = ["Scheduled"]
+            result = self.requestClient().getRequestIDsList(statusList, self.DEL_LIMIT)
+            if not result["OK"]:
+                self.log.error("Failed to get list of Scheduled requests:", result["Message"])
+                return result
+            requestIDsList = result["Value"]
+            cancelled = 0
+            for requestID, status, lastUpdate in requestIDsList:
+                if lastUpdate < cancelTime:
```

Comment:
```suggestion
                if lastUpdate < (datetime.datetime.utcnow() - datetime.timedelta(days=self.cancelGraceDays)):
```

Reply:
I took the code, but left it as a variable, there's no shortage of screen space and I find it easier to read.